app/bower.json

```diff
@@ -26,6 +26,6 @@
   },
   "resolutions": {
     "jquery": "1.9.1 - 2",
-    "angular": "1.5.8"
+    "angular": ">=1.4.0"
```

Comment:
odd...   So, in CircleCI I was getting errors, the ./depends.sh script was puking on Angular.  So then locally I did a `bower install -f` and that made it work.   Instead of leaving that in the depends.sh script, I saw it changed the bower.json file, so I checked that in, and then CircleCI started passing!